### PR TITLE
[script.module.dropbox_auth] 1.0.0

### PR DESCRIPTION
**Pull request by SchapplM**
### Description
New addon: Provide [authentification for dropbox API to other addons via QR code](https://kodi.wiki/view/Add-on:Dropbox#Method_A)
* [code was already in the addon script.module.dropbox, but removed there](https://github.com/micahg/script.module.dropbox/commit/b1eff513532ee367232e63e123fb2c57eb3b51e9#commitcomment-38740357)
* moved code from [pre-matrix script.module.dropbox](https://github.com/micahg/script.module.dropbox) to [this script.module.dropbox_auth](https://github.com/SchapplM/xbmc-addon-dropboxauth)
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0
### Dependencies
depends on script.module.qrcode. Merge #1495 first.

### CODE REVIEW

**Comment by SchapplM** [comment]
Accidently had one additional commit in the first try. Did force push.

**Comment by SchapplM** [comment]
From my side this is finished. Do you have some comments? Otherwise I would squash the commits.
If you need another addon using it, I can also start the pull request for [service.watchedlist](https://github.com/SchapplM/xbmc-addon-service-watchedlist/tree/python3).

**Comment by enen92** [comment]
Please submit any add-ons that require your already submitted dependencies. Normally we only review them when there is actually an add-on depending on them

File: script.module.dropbox_auth/addon.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <import addon="script.module.requests" version="2.22.0+matrix.1" />
+    <import addon="script.module.six" version="1.14.0+matrix.1" />
+    <import addon="script.module.qrcode" version="6.1.0+matrix.2" /> <!-- For QR Code display for Dropbox API authentication -->
+  </requires>
+  <extension point="xbmc.python.library" library="authenticate.py" /> <!-- entry to start the authentification script from the settings dialog of other addons -->
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Dropbox Auth API</summary>
+    <description lang="en_GB">Provide Dropbox Authentification</description>
+    <platform>all</platform>
+    <language/>
```
Comment (enen92) [inline script.module.dropbox_auth/addon.xml:18]: please remove language tag as the addon doesn't provide content

File: script.module.dropbox_auth/resources/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+# Dummy file to make this directory a package.
```
Comment (enen92) [inline script.module.dropbox_auth/resources/__init__.py:1]: shouldn't be needed anymore

**Comment by SchapplM** [comment]
I implemented the comments

**Pull request merged**